hydra-node/test/Hydra/Model.hs

```diff
@@ -719,7 +742,7 @@ waitForUTxOToSpend utxo key value node = go 100
       pure $ Left utxo
     n -> do
       node `send` Input.GetUTxO
-      threadDelay 5
+      threadDelay 0.1
```

Comment:
This will make the model runner spam / poll the node quite a lot. Do we need this change?

Reply:
No, this is a leftover